blueprints/serverless/cloud-run-explore/main.tf

```diff
@@ -0,0 +1,163 @@
+locals {
```

Comment:
You're missing the header/boilerplate in tf files

Reply:
I know, I wanted to go step by step (and break things) to get familiarized.